task/sast-shell-check/0.1/sast-shell-check.yaml

```diff
@@ -121,7 +121,12 @@ spec:
             echo "KFP_GIT_URL is not set. Skipping false positive filtering."
         fi
 
-        csgrep --mode=json --event "$CSGREP_EVENT_FILTER" "$OUTPUT_FILE"
+        if [[ "$IMP_FINDINGS_ONLY" == "true" ]]; then
+            CSGREP_EVENT_FILTER='\[SC(1020|1035|1054|1066|1068|1073|1080|1083|1099|1113|1115|1127|1128|1143|2043|2050|\
+                                      2055|2057|2066|2069|2071|2077|2078|2091|2092|2157|2171|2193|2194|2195|2215|2216|\
+                                      2218|2224|2225|2242|2256|2258|2261)\]$'
```

Comment:
@rhyw You cannot write it like this because the leading spaces would be included in the regular expression passed to the `--event` option of `csgrep`.